client/src/components/VoterRequirements.jsx

```diff
@@ -1,69 +1,175 @@
+        not need the card with you in order to vote on election day! For more
+        information about what you can expect when you go to vote, please see
+        our page about <Link to="/election-day">Election Day</Link>.
+      </li>
     </ul>
     <h2 className="VoterRequirements-heading">Need other information?</h2>
-    <p>If you’re not a first-time voter, you may have different questions - hopefully we can help! </p>
-    <h3 className="VoterRequirements-heading">Changing Registration Information</h3>
-    <p>The North Carolina Voter Registration Application may be used to change any voting information, including: name, address and party affiliation. The change notification must be signed, and should be sent to the appropriate County Board of Elections by the voter registration deadline (25 days before each election).</p>
-    <p>Once you have completed and mailed your changes to your local board of elections, allow two to six weeks for delivery of your voter identification card. Read the information thoroughly and note any changes or mistakes on the card. A voter identification card is for your use only, you do not need the card in order to vote.</p>
-    <h3 className="VoterRequirements-heading">Military & Overseas Citizens Voting Information</h3>
-    <p>Some citizens, such as people serving in the military who are away from their permanent home on Election Day, their families, or U.S. citizens who are living abroad, have special voting rights and ways to register to vote. </p>
-    <p>If this describes you, you have the choice of either requesting a mail-in absentee ballot the same way as other registered voters, or applying to register and/or vote through special programs for military and overseas voters as described here.</p>
-    <h3><a href="https://ncvoter.org/registering-to-vote/">For more information visit this page from Democracy NC</a></h3>
+    <p>
+      If you’re not a first-time voter, you may have different questions -
+      hopefully we can help!{" "}
+    </p>
+    <h3 className="VoterRequirements-heading">
+      Changing Registration Information
+    </h3>
+    <p>
+      The North Carolina Voter Registration Application may be used to change
+      any voting information, including: name, address and party affiliation.
+      The change notification must be signed, and should be sent to the
+      appropriate County Board of Elections by the voter registration deadline
+      (25 days before each election).
+    </p>
+    <p>
+      Once you have completed and mailed your changes to your local board of
+      elections, allow two to six weeks for delivery of your voter
+      identification card. Read the information thoroughly and note any changes
+      or mistakes on the card. A voter identification card is for your use only,
+      you do not need the card in order to vote.
+    </p>
+    <h3 className="VoterRequirements-heading">
+      Military & Overseas Citizens Voting Information
```

Comment:
Prefer to use HTML escaped ampersand (`&amp;`) over the regular old ampersand (`&`).  Or just spell out "and" to avoid the issue altogether.

I might be wrong altogether since this is a `.jsx` file, FWIW.